Samsung/TizenRT
# Add WifiCSI Example

To test CSI:
Ensure that wifi is connected using TASH command
Enter TASH command **wificsi** to run this example.
